### ISSUE TITLE
Missing property "groups" for Runners

### ISSUE THREAD

**Issue by ozeebee** [jdalrymple/gitbeaker#3347]
**Description**

- Node.js version: 18
- Gitbeaker version: 39.5.1
- Gitbeaker release (cli, node, browser, core, requester-utils): rest
- OS & version: macOS

**Steps to reproduce**

Get a runner's details.  
It returns an `ExpandedRunnerSchema` object without the `groups` property (even if the `runner_type` is `group_type`)

**Expected behaviour**

Runner object should have a `groups` array.

**Actual behaviour**

no `groups` array is in the schema even though GitLab API returns it.  
Tested on a GitLab 16.1 instance

**Possible fixes**

make it an optional property if not all gitlab versions are supporting it.

**Checklist**
- [x] I have checked that this is not a duplicate issue.
- [x] I have read the documentation.

**Comment by jdalrymple** [comment]
This is simply because they didnt include it in their docs which is what i base the schema off of. Ill double check and add it in

**Comment by ozeebee** [comment]
Sure, here'a redacted sample response:
```json
{
  "id": 1886,
  "description": "my runner",
  "ip_address": "11.22.33.44",
  "active": true,
  "paused": false,
  "is_shared": false,
  "runner_type": "group_type",
  "name": null,
  "online": true,
  "status": "online",
  "tag_list": [
    "tag1",
    "tag2"
  ],
  "run_untagged": false,
  "locked": false,
  "maximum_timeout": null,
  "access_level": "not_protected",
  "version": "16.1.0",
  "revision": "b72e108d",
  "platform": "linux",
  "architecture": "amd64",
  "contacted_at": "2023-07-13T08:02:03.528Z",
  "projects": [],
  "groups": [
    {
      "id": 33,
      "web_url": "https://mygitlabserver.com/groups/my-group",
      "name": "MyGroup"
    }
  ]
}
```

Thanks !

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`39.7.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/39.7.0) :rocket:

**Comment by ozeebee** [comment]
Thanks, I confirm this works !
